horovod/run/common/util/safe_shell_exec.py

```diff
@@ -84,7 +84,7 @@ def forward_stream(src_fd, dst_stream, prefix, index):
                     line_buffer = ''
 
 
-def execute(command, env=None, stdout=None, stderr=None, index=None, event=None):
+def execute(command, env=None, stdout=None, stderr=None, index=None, events=None):
```

Comment:
can we rename

    def set_sigterm_received(signum, frame):

to

    def set_event_on_signal(signum, frame):

in this file as we did in `horovod/run/gloo_run.py`?

Reply:
Done.